VideoWeb/VideoWeb/ClientApp/src/app/services/conference/video-control-cache.service.ts

```diff
@@ -18,6 +18,11 @@ export class VideoControlCacheService {
         private logger: LoggerService
     ) {
         this.conferenceService.currentConference$.subscribe(conference => {
+            if (!conference) {
+                this.logger.warn(`${this.loggerPrefix} No conference loaded. Skipping loading of hearing state for conference`);
+                return;
+            }
+
             this.storageService.loadHearingStateForConference(conference.id).subscribe(state => {
                 this.hearingControlStates = state;
             });
```

Comment:
I know this is existing code. But does this not create a new subscription every time the currentConference changes that never completes?

Reply:
Well spotted. Yeah, overlooked as it should be a call to an endpoint but currently it is using the cache. Will add a pipe.